### PR TITLE
Use $GALAXY_SLOTS in kc-align instead of threads as a tool param

### PR DESCRIPTION
**Pull request by natefoo**
Also set the version to the full tool version.

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by natefoo** [comment]
Not sure if I broke the tests and, if so, how I figure out what's broken?

File: tools/kc-align/kc-align.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
                 -d '$position.dist'
             #end if
             $compress
-            -th '$threads'
+            -th '\${GALAXY_SLOTS:-1}'
```
Comment (nsoranzo) [inline tools/kc-align/kc-align.xml:20]:
```suggestion
            -th \${GALAXY_SLOTS:-1}
```

**Comment by nsoranzo** [comment]
> Not sure if I broke the tests and, if so, how I figure out what's broken?

Yeah, you did :laughing: , see comment above for a fix. For future reference, you can inspect the planemo report by downloading the "All tool test results" artifact.

**Comment by bernt-matthias** [comment]
Excellent idea. Lets do the same for 

- tools/mummer4/mummer.xml (params `threads`, `qthreads`)
- tools/colibread/lordec.xml (param -`T`)

**Comment by natefoo** [comment]
@nsoranzo Ah, I saw the message about downloading artifacts but couldn't find the Artifacts link - found it. Thanks!

**Pull request merged**